plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcConnectorTest.java

```diff
@@ -45,4 +46,10 @@ protected Session withSmallRowGroups(Session session)
                 .setCatalogSessionProperty("iceberg", "orc_writer_max_stripe_rows", "10")
                 .build();
     }
+
+    @Override
+    public void testInsertRowConcurrently()
+    {
+        throw new SkipException("TODO Enable this test after fixing concurrent INSERT issue");
```

Comment:
What's the issue?

Reply:
It was incorrect issue. Should I file an issue? I guess it comes from File HMS because we have `io.trino.tests.product.iceberg.TestIcebergInsert`.

Reply:
is it solved by #12026's changes? cc @alexjo2144

Reply:
It was resolved as far as tried locally. Removed the overridden tests.